### PR TITLE
Drops "type" from "script", "link" and "style" tags

### PR DESCRIPTION
**Pull request by newbazz**
Closes https://github.com/plone/Products.CMFPlone/issues/1078

### CODE REVIEW

**Comment by newbazz** [comment]
@hvelarde changes done

File: CHANGES.rst [hunk 1]
```diff
@@ -73,6 +73,8 @@ Bug fixes:
 - Fixes #2105: how to get ``email_from_name`` information in sendto_form.
   [cekk]
 
+- Remove ``type`` from ``style`` and ``link`` tag.
```
Comment (hvelarde) [inline CHANGES.rst]: you need to add your name below this entry: `[newbazz]`

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/resources/browser/scripts.pt [hunk 2]
```diff
@@ -10,16 +10,14 @@
...
-  <script type="text/javascript"
-      tal:attributes="
-        src script/src;
-        data-bundle script/bundle;
-        async script/async|nothing;
-        defer script/defer|nothing"/>
+  <script tal:attributes="src script/src;
+                          data-bundle script/bundle;
+                          async script/async|nothing;
+                          defer script/defer|nothing"/>
```
Comment (sarn0ld) [inline Products/CMFPlone/resources/browser/scripts.pt]: This line should be `defer script/defer|nothing" ></script>` to fix #2870

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/resources/browser/scripts.pt (hunk 2 above)
Comment (thet) [reply Products/CMFPlone/resources/browser/scripts.pt]: I missed that - yes that's necessary. See: https://github.com/plone/Products.CMFPlone/pull/2871

**Comment by jensens** [comment]
There was no activity on this for a while. I close this request for now. If you plan to work further on this please reopen and go on.

**Pull request closed without merging**